README.md

```diff
@@ -1,103 +1,122 @@
-# Logstash Plugin
+# Logstash Dynatrace output plugin
 
 [![Travis Build Status](https://app.travis-ci.com/dynatrace-oss/logstash-output-dynatrace.svg)](https://app.travis-ci.com/dynatrace-oss/logstash-output-dynatrace)
 
 > This project is developed and maintained by Dynatrace R&D.
 
-A [Logstash](https://github.com/elastic/logstash) output plugin for sending logs to the Dynatrace [Generic log ingest API v2](https://www.dynatrace.com/support/help/how-to-use-dynatrace/log-monitoring/log-monitoring-v2/post-log-ingest/).
-
-## Documentation
-
-Logstash provides plugin documentation in a [central location](https://www.elastic.co/guide/en/logstash/current/plugins-outputs-dynatrace.html).
-
-## Developing
-
-### 1. Plugin Developement and Testing
-
-#### Code
+- [Installation](#installation)
+- [Example Configuration](#example-configuration)
+- [Configuration Overview](#configuration-overview)
+  - [Common Options](#common-options)
+- [Configuration Detail](#configuration-detail)
+  - [`ingest_endpoint_url`](#ingest_endpoint_url)
+  - [`api_key`](#api_key)
+  - [`ssl_verify_none`](#ssl_verify_none)
+  - [`codec`](#codec)
+  - [`enable_metric`](#enable_metric)
+  - [`id`](#id)
 
-- To get started, you'll need JRuby with the Bundler gem installed.
-
-- Clone the plugin from github
+A [Logstash](https://github.com/elastic/logstash) output plugin for sending logs to the Dynatrace [Generic log ingest API v2](https://www.dynatrace.com/support/help/how-to-use-dynatrace/log-monitoring/log-monitoring-v2/post-log-ingest/).
+Please review the documentation for this API before using the plugin.
+## Installation
 
-- Install dependencies
+Logstash is typically installed in the `/usr/share/logstash` directory, and plugins are installed using the `/usr/share/logstash/bin/logstash-plugin` command.
+If your logstash installation directory is different than this, your `logstash-plugin` command may be in a different location.
 
 ```sh
-bundle install
+/usr/share/logstash/bin/logstash-plugin install logstash-output-dynatrace
 ```
 
-#### Test
+## Example Configuration
 
-- Update your dependencies
+See below for a detailed explanation of the options used in this example configuration.
 
-```sh
-bundle install
+```
```

Comment:
```suggestion
```ruby
```

Reply:
This is not ruby. I can add the language signifier anyway just to get some level of syntax highlighting since it is at least similar to ruby though?

Reply:
Yeah, thats what I was going for. I think it looks better if you put the language specifier there.